### PR TITLE
EPMRPP-63628: log view all launches label

### PR DESCRIPTION
**Pull request by oranje322**

### CODE REVIEW

**Review by chivekrodis — APPROVED** [review]

File: app/src/pages/inside/logsPage/logToolbar/logToolbar.scss [hunk 1]
```diff
@@ -35,3 +35,13 @@
 .left-arrow-button {
   margin-right: -1px;
 }
+
+.breadcrumbs-wrapper {
+  max-width: 71%;
+}
+
+@media (max-width: $SCREEN_XL_MAX) {
```
Comment (AmsterGet) [inline app/src/pages/inside/logsPage/logToolbar/logToolbar.scss:43]: Please remove it. Not need to add new screen size breakpoint.

File: app/src/pages/inside/logsPage/logToolbar/logToolbar.jsx [hunk 2]
```diff
@@ -168,19 +173,25 @@ export class LogToolbar extends Component {
...
           {logViewMode === DETAILED_LOG_VIEW ? (
             <>
               {!debugMode && (
                 <InputCheckbox onChange={this.changeHistoryLineMode} value={includeAllLaunches}>
-                  {intl.formatMessage(messages.historyAllLaunchesLabel)}
+                  {intl.formatMessage(
+                    window.innerWidth > SCREEN_XL_MAX
+                      ? messages.historyAllLaunchesLabel
+                      : messages.historyShortAllLaunchesLabel,
+                  )}
```
Comment (AmsterGet) [inline app/src/pages/inside/logsPage/logToolbar/logToolbar.jsx:194]: As we need to use short label only on mobile, please use corresponding `SCREEN_XS_MAX` breadkpoint.

Comment (AmsterGet) [reply app/src/pages/inside/logsPage/logToolbar/logToolbar.jsx:194]: Also you may provide a custom class to `InputCheckbox` component to control wrappig behaviour. F.e. adding the CSS property `white-space: nowrap;` to this field will always keep all words in one line.

File: app/src/pages/inside/logsPage/logToolbar/logToolbar.scss [hunk 3]
```diff
@@ -35,3 +35,13 @@
 .left-arrow-button {
   margin-right: -1px;
 }
+
+.breadcrumbs-wrapper {
```
Comment (AmsterGet) [inline app/src/pages/inside/logsPage/logToolbar/logToolbar.scss:39]: After providing custom class to the `InputCheckbox` component it will be unnecessary.

**Review by AmsterGet — CHANGES_REQUESTED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**